starlite/cache/memcached_cache_backend.py

```diff
@@ -80,8 +80,7 @@ async def set(self, key: str, value: Any, expiration: int) -> None:  # pylint: d
             None
         """
 
-        await self._memcached_client.set(key.encode(), self._config.serialize(value), exptime=expiration)
-        return None
+        await self._memcached_client.set(key.encode(), self._config.serialize(value), expiration=expiration)
```

Comment:
@Goldziher this change will not work with `aiomcache` - the parameter name is `exptime`:
https://github.com/aio-libs/aiomcache/blob/20f4b18d5fc4c54047ebaf6d28e6024d80d591e4/aiomcache/client.py#L252